ColabTool/Shared/Post/index.jsx

```diff
@@ -1,20 +1,46 @@
 import React, { PropTypes } from 'react';
 import {
   Card,
+  LinkifiedText,
 } from '@bufferapp/components';
 import style from './style.css';
 import PostFooter from '../PostFooter';
 import PostHeader from '../PostHeader';
 import RetweetPanel from '../RetweetPanel';
 
-const renderRetweetPanel = (retweetProfile) => {
+const renderRetweetComment = (comment, links) => (
+  <LinkifiedText
+    links={links}
+    newTab
+    size={'mini'}
+    unstyled
+  >
+    { comment }
+  </LinkifiedText>
+);
+
+const renderRetweet = (children, retweetComment, retweetProfile) => {
   if (retweetProfile) {
-    return (
-      <div className={style['retweet-profile-wrapper']}>
-        <RetweetPanel {...retweetProfile} />
-      </div>
-    );
+    if (retweetComment) {
+      return (
```

Comment:
Heya @stevemdixon 😄  Thanks so much for your work on this!
I think you can remove this second conditional checking for `retweetComment` since you're checking that in the  ternary on line 27.